nautobot/dcim/templates/dcim/cable_connect.html

```diff
@@ -229,5 +232,16 @@ <h3>
             $('#id_termination_b_id').find("option").remove();
         });
     });
+
+    // clear the select2 widget for the module when a new device is selected
+    // jQuery required for select2
+    document.addEventListener("DOMContentLoaded", function() {
+        $('select#id_termination_b_device').select2();
+        $('select#id_termination_b_module').select2();
```

Comment:
Is this necessary? I would have thought it'd already be done automatically. Just an order of operations issue?

Reply:
This is for the case where you've already selected a device and module but you change your mind and want to select a different device. This will clear out the module when you select a new device. Is there a way to make that automatic?

Reply:
Sorry, you mean the `select2()` initialization? I ran into race conditions before where `DOMContentLoaded` is initializing the select2 component at the same time as this event listener is firing. That's why I'm doing it explicitly here.

Reply:
Yeah, that's what I meant. Sounds reasonable.